src/components/HeroStrategy/hero-right.tsx

```diff
@@ -96,17 +113,7 @@ export const HeroStrategyRight: VFC<HeroStrategyRightProps> = ({
         </>
       ) : (
         <>
-          <BaseButton
-            w="full"
-            h="50px"
-            onClick={() => {
-              analytics.track("strategy.buy-sell", {
-                strategyCard: cellarData.name,
-                landingType: landingType(),
-              })
-              buyOrSellModal.onOpen()
-            }}
-          >
+          <BaseButton w="full" h="50px" onClick={handleBuyOrSell}>
```

Comment:
this button and "View Details" button will be appeared. Buy or sell modal button trigger should be hided if there's no exchanges.

Reply:
Sam alraedy dicuss about this with Yoni, and they want to display both of these button. because it will be confusing to just display one button in the real yield usd. but both button will have same functionality